src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
```

Comment:
Use abstract references instead of specific ones where possible to take advantage of polymorphism. For example, `Machine truck = new Truck();` allows you to use the `Machine` type for all your objects.